tests/utils.py

```diff
@@ -854,7 +856,7 @@ def clear(self):
 
 
 @contextmanager
-def snapshot_context(token, ignores=None, tracer=None, async_mode=True, variants=None):
+def snapshot_context(token, ignores=None, tracer=None, async_mode=True, variants=None, wait_for_traces=None):
```

Comment:
Am I correct in understanding `wait_for_traces` is also specifying the number of traces that we're expecting? If so, I'm not so sure this is the best name since it implies a boolean. Maybe `traces_to_expect`? 😕

Reply:
yeah, crossed my mind too. I'll update it.